cmgd/cmgd_frntnd_adapter.c

```diff
@@ -1777,3 +1854,34 @@ void cmgd_frntnd_adapter_status_write(struct vty *vty, bool detail)
 	vty_out(vty, "  Total: %d\n", 
 		(int) cmgd_frntnd_adptr_list_count(&cmgd_frntnd_adptrs));
 }
+
+void cmgd_frntnd_adapter_perf_measurement(struct vty *vty, bool config)
+{
+	cmgd_frntnd_client_adapter_t *adptr;
+	cmgd_frntnd_sessn_ctxt_t *sessn;
+
+	FOREACH_ADPTR_IN_LIST(adptr) {
```

Comment:
Same comment as before. Just have a global. Not per adapter. Maybe inside cmd_master.